lib/nconf-truth.ts

```diff
@@ -20,7 +20,7 @@ nconf.Env.prototype.loadEnv = function() {
       }
     })
     .forEach(function(key) {
-      var value = process.env[key];
+      var value: string | undefined | boolean = process.env[key];
```

Comment:
I thought ```process.env``` may only contain ```strings | undefined```?
https://nodejs.org/dist/latest-v8.x/docs/api/process.html#process_process_env

Reply:
It can, but the code reassigns the local (`var`) to a boolean lower down; this is its entire purpose, so the temporary lie is valid, imo.